src/agent/debuglet.ts

```diff
@@ -660,11 +660,14 @@ export class Debuglet extends EventEmitter {
    * @private
    */
   removeBreakpoint_(breakpoint: Breakpoint): void {
+    const that = this;
     this.logger_.info('\tdeleted breakpoint', breakpoint.id);
     // TODO: Address the case when `breakpoint.id` is `undefined`.
     delete this.activeBreakpointMap_[breakpoint.id as string];
     if (this.v8debug_) {
-      this.v8debug_.clear(breakpoint);
+      this.v8debug_.clear(breakpoint, function(err) {
```

Comment:
You should be able to use an arrow function here so that you can use `this` directly within the arrow function without having to use `const that = this;`.